src/components/MovieTitle/MovieTitle.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
```

Comment:
I guess the name of the folder and the file should be `Movie Tile`, not `Movie Title`.